android/src/main/java/io/branch/rnbranch/RNBranchModule.java

```diff
@@ -117,7 +122,8 @@ public static void reInitSession(Activity reactActivity) {
         Intent intent = reactActivity.getIntent();
         if (intent != null) {
             intent.putExtra("branch_force_new_session", true);
-            branch.reInitSession(reactActivity, referralInitListener);
+            notifyJSOfInitSessionStart(reactActivity, intent.getData());
```

Comment:
Is the name `notifyJSOfInitSessionStart` misspelled?

Reply:
I don't think it is. This is a camel-case representation of: "notify JS of init session start." But it could be called something else.

Reply:
haha I see! Yeah, maybe we can leave a comment above the method "notify JS of init session start."

I personally would name it `notifyJSInitSessionStarted` or `notifyJSThatInitSessionStarted` but naming is always subjective, and you work on the RN project most of the time anyway, so whatever you choose is fine by me :)